src/main/java/io/github/eliux/mega/cmd/ExportInfo.java

```diff
@@ -52,4 +80,13 @@ public String getRemotePath() {
     public String getPublicLink() {
         return publicLink;
     }
+
+    public LocalDate getExpireDate() {
+        return expireDate.map(s -> LocalDateTime.parse(s, DateTimeFormatter.ofPattern(dateFormat, Locale.US)).toLocalDate()).orElse(null);
```

Comment:
First of all it is an antipattern to use `.orElse(null)`. The value `null` is like Voldemort, "the one who must not be named". Instead we use Optional to indicate that such value could not be present.

In addition, you should not do the parsing every time this attribute is read. or you will have to do it too many times. It is better doing it when the value is set as explained before. 
So this getter would be a simple one:

```
public LocalDate getExpireDate() {
  return expireDate;
}
```